pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -73,6 +75,22 @@ const (
 	clusterImageSetFoundReason            = "ClusterImageSetFound"
 )
 
+var (
+	deprovisionJobsDurationHistogram = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: "deprovision_jobs_duration_seconds",
+			Help: "Deprovision Jobs duration distribution",
+			//Buckets: []float64{0.5, 1, 2, 4, 8, 16},
+			Buckets: prometheus.ExponentialBuckets(1, 2, 9),
```

Comment:
Can you help me understand these values and how this works?

Reply:
It creates buckets with the start value 1, step 2 and count 9, so basically buckets that correspond to <1 seconds to <256 seconds

Reply:
Hmm interesting, could you get us three buckets, jobs less than 60 seconds, jobs less than 10 minutes, and anything over that?

Reply:
yes definitely